packages/core/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@manypkg/core",
+  "version": "1.0.0",
+  "license": "MIT",
+  "main": "dist/get-packages.cjs.js",
```

Comment:
this looks incorrect for this package - you can probably call `preconstruct fix` from the root autoupdate this to the correct value